### PR TITLE
adding trainer tests

### PR DESCRIPTION
**Pull request by GolamCh**

### CODE REVIEW

File: src/test/java/com/sparta/eng82/tests/stepdefs/SharedStepDefs.java [hunk 1]
```diff
@@ -13,31 +13,31 @@
...
-
-    @When("I click on {string} home page button")
-    public void iClickOnHomePageButton(String arg0) {
-
-    }
-
-    @Then("I go to the {string}home page")
-    public void iGoToTheHomePage(String arg0) {
-    }
+//    @Given("I am logged in as an {string}")
```
Comment (alexsikorski) [inline src/test/java/com/sparta/eng82/tests/stepdefs/SharedStepDefs.java]: Uncomment all of this please!

**Review by jackingham — APPROVED** [review]
good work

**Review by alexsikorski — CHANGES_REQUESTED** [review]
Uncomment shared stepdef please

**Comment by jackingham** [comment]
> Uncomment shared stepdef please

already done please close issue

**Review by AJMalcolm — APPROVED** [review]

**Review by alexsikorski — APPROVED** [review]
Good job

**Pull request merged**